src/main/java/controller/RequestController.java

```diff
@@ -0,0 +1,123 @@
+package controller;
+
+import db.DataBase;
+import http.HttpQueryParams;
+import http.HttpRequest;
+import http.HttpResponse;
+import model.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import utils.FileIoUtils;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+public class RequestController {
+    private static final Logger logger = LoggerFactory.getLogger(RequestController.class);
+
+    public static void handleRequest(HttpRequest httpRequest, DataOutputStream dos) throws IOException {
+        String path = httpRequest.getStartLine().getPath();
+        if (isFile(path)) {
+            handleFile(httpRequest, dos);
+            return;
+        }
+
+        if (path.startsWith("/user/create")) {
+            if (httpRequest.getStartLine().getMethod().equals(HttpMethod.GET)) {
+                handleUserCreate(httpRequest.getQueryParams(), dos);
+                return;
+            }
+            if (httpRequest.getStartLine().getMethod().equals(HttpMethod.POST)) {
+                String body = (String) httpRequest.getBody();
+
+                HttpQueryParams queryParams = new HttpQueryParams();
+                for (String line : body.split("&")) {
+                    String[] tokens = line.split("=");
+                    queryParams.put(tokens[0], tokens[1]);
+                }
```

Comment:
이 케이스 또한 `HttpQueryParams`의 정적 팩토리 메서드로 만들어봐도 좋을 것 같아요. 🙂

```suggestion
                HttpQueryParams queryParams = HttpQueryParams.xxx(body);
```